docs/catalog/cloud/use-compiled-languages.md

```diff
@@ -0,0 +1,44 @@
+---
+version: 1.0
+submitted_by: markus-ntt-seidl
+published_date: TBD
+category: cloud
+tags: 
+ - cloud
+ - role:cloud-engineer
+ - size:small
+---
+
+# Use compiled languages
+
+## Description
+
+Interpreted languages need to be parsed, compiled and executed when the application starts or a workload arrives. This tends to be more energy heavy then when a compiled language is used. The compilation is then only done once, saving on resources.
+
+## Solution
+
+Use compiled languages whenever possible or compile interpreted languages.
+
+## SCI Impact
+
+`SCI = (E * I) + M per R`  
+[Software Carbon Intensity Spec](https://grnsft.org/sci)
+
+Concerning the SCI equation, use compiled languages will impact as follows:
+
+- `E`: The embodied carbon emissions will be reduced, as compiled binaries tend to be smaller then the sources they are compiled from
```

Comment:
Did you mean to use `M` instead of `E` here?  The description points to embodied carbon but I also see an energy use drop with compiled languages since the compilation is done once instead of at each request.

Reply:
True.